javascript-carousel/main.js

```diff
@@ -0,0 +1,81 @@
+var $image = document.querySelector('img');
+var imageSrcValues = ['025.png', '004.png', '007.png', '039.png', '001.png'];
+var indexOfImage = 0;
+var intervalIndex = 0;
+var $buttonCircles = document.querySelector('.circle-buttons-container');
+var $circles = document.querySelectorAll('.fa-circle');
+var $leftArrow = document.querySelector('.fa-angle-left');
+var $rightArrow = document.querySelector('.fa-angle-right');
+
+function changeImage() {
+  $image.setAttribute('src', 'images/' + imageSrcValues[intervalIndex]);
+  matchIntervalToCircle();
+  intervalIndex++;
+  if (intervalIndex === 5) {
+    intervalIndex = 0;
+  }
+}
+
+setInterval(changeImage, 3000);
```

Comment:
The countdown to automatically switch images should start over when the user manually changes it:

---

![image](https://user-images.githubusercontent.com/16143771/163262217-754925f3-0f2c-49ec-ac5e-d77134429560.png)